### PR TITLE
gatsby: textpage pro desktop

### PR DESCRIPTION
**Pull request by vasimi**
textpage = detail opatreni apod.

### CODE REVIEW

File: gatsby/src/components/text-box/text-box.tsx [hunk 1]
```diff
@@ -2,28 +2,46 @@ import React from 'react';
...
+        </p>
+        <p>
+          <Link label="Internal" to="/" />
+        </p>
+        <h2>Odkazy a zdroje</h2>
+        <p>
+          <Link label="www.szu.cz/hygiena" to="https://www.szu.cz/hygiena" />
+        </p>
+        <p>
+          <Link label="www.vlada.cz" to="https://vlada.cz" />
```
Comment (fidelman) [inline gatsby/src/components/text-box/text-box.tsx:38]: maybe we can set `label` as an optional and use `to` as a `label` if it is undefined?

File: gatsby/src/components/text-box/text-box.tsx [hunk 2]
```diff
@@ -2,28 +2,46 @@ import React from 'react';
...
-        <Link label="External" to="https://search.seznam.cz" />
-      </p>
-      <p>
-        <Link label="Internal" to="/" />
-      </p>
-      <p><em>Italic paragraph last updated 1 hour ago</em></p>
-    </div>
+    <Row className={styles.textBox}>
+      <Col colLg={8}>
+        <h2>Headline #2</h2>
```
Comment (fidelman) [inline gatsby/src/components/text-box/text-box.tsx:12]: how are we gonna parse this content from BE? Will we receive HTML string and parse it as `dangerouslySetInnerHTML `?

**Review by fidelman — APPROVED** [review]
Great job  👍 left a few questions

File: gatsby/src/components/text-box/text-box.tsx (hunk 1 above)
Comment (vasimi) [reply gatsby/src/components/text-box/text-box.tsx:38]: Nope, we were discussing this with BE and they'll always send `label`.

File: gatsby/src/components/text-box/text-box.tsx (hunk 2 above)
Comment (vasimi) [reply gatsby/src/components/text-box/text-box.tsx:12]: Probably, dunno yet. I wanted to be prepared for that so I didn't use any classes.

**Pull request merged**